test/functional/helpers/spectrum.js

```diff
@@ -237,7 +240,17 @@ const componentWrappers = {
     return {
       async expectTitle(title) {
         await switchToIframe();
-        await selector.find(".spectrum-Alert-header").withText(title);
```

Comment:
This is merely a selector, so we weren't ever doing an assertion that an element matching the selector actually exists, which was allowing the test to pass when it shouldn't have.

Reply:
Thanks